test/chemistry/test_qeom_vqe.py

```diff
@@ -43,11 +46,13 @@ def setUp(self):
             pyscf_driver = PySCFDriver(atom=atom,
                                        unit=UnitsType.ANGSTROM, charge=0, spin=0, basis='sto3g')
             self.molecule = pyscf_driver.run()
+            warnings.filterwarnings('ignore', category=DeprecationWarning)
```

Comment:
Similar comment to EomEE test above

Reply:
this will come with the new Excited States PR